src/components/Form/index.tsx

```diff
@@ -188,245 +188,302 @@ export const Form = ({
       'relationship:same-as'
     ],
     howKnown: [
-      'first_hand',
-      'second_hand',
-      'website',
-      'verified_website',
-      'verified_login',
-      'signed_claim',
-      'blockchain',
-      'physical_document',
-      'integration'
+      'First hand',
+      'Second hand',
+      'Website',
+      'Verified website',
+      'Verified login',
+      'Signed claim',
+      'Blockchain',
+      'Physical document',
+      'Integration'
     ]
   }
 
   return (
     <>
-      <DialogTitle>
```

Comment:
you made changes in the function it self , please change only in the design and text

Reply:
when i was to remove conflict so this happens i will correct it